coalib/results/Diff.py

```diff
@@ -166,16 +166,20 @@ def from_clang_fixit(cls, fixit, file):
         """
         assert isinstance(file, (list, tuple))
```

Comment:
The assert statement can throw `AssertionError`. Seems we can have it in the `try` branch.

Reply:
@yashasingh  I totally agree with you, but I think handling errors due to Assertion would be a different issue and this PR's goal is to detect unsupported clang and report a useful error message. Maybe we can have a different issue for catching AssertionError?

Reply:
Actually this is intended without a try block. The idea is that we want to see the error and making coala stop, otherwise we wouldn't use an `assert` here ;) Otherwise it's the same like doing:
```
try:
    raise Exception
except Exception:
    pass
```